packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -57,12 +67,93 @@ class ContainerPlugin {
           name: item.name || undefined,
         }),
       ),
+      runtimePlugins: options.runtimePlugins,
     };
   }
 
+  // container should not be affected by splitChunks
+  static patchChunkSplit(compiler: Compiler, name: string): void {
+    const { splitChunks } = compiler.options.optimization;
+    const patchChunkSplit = (cacheGroup: CacheGroup) => {
+      switch (typeof cacheGroup) {
+        case 'boolean':
+        case 'string':
+        case 'function':
+          break;
+        //  cacheGroup.chunks 会继承 splitChunks.chunks ，因此只需要对单独设置了 chunks 的 做修改
+        case 'object':
+          {
+            if (cacheGroup instanceof RegExp) {
+              break;
+            }
+            if (!cacheGroup.chunks) {
+              break;
+            }
+            if (typeof cacheGroup.chunks === 'function') {
+              const prevChunks = cacheGroup.chunks;
+              cacheGroup.chunks = (chunk) => {
+                if (chunk.name && chunk.name === name) {
+                  return false;
+                }
+                return prevChunks(chunk);
+              };
+              break;
+            }
+
+            if (cacheGroup.chunks === 'all') {
+              cacheGroup.chunks = (chunk) => {
+                if (chunk.name && chunk.name === name) {
+                  return false;
+                }
+                return true;
+              };
+              break;
+            }
+            if (cacheGroup.chunks === 'initial') {
+              cacheGroup.chunks = (chunk) => {
+                if (chunk.name && chunk.name === name) {
+                  return false;
+                }
+                return chunk.isOnlyInitial();
+              };
+              break;
+            }
+          }
+          break;
+      }
+    };
+
+    if (!splitChunks) {
+      return;
+    }
+    // 修改 splitChunk.chunks
```

Comment:
It needs to be changed into English, as well as the rest of the following content

Reply:
got